src/Stratis.Features.FederatedPeg/SourceChain/DepositExtractor.cs

```diff
@@ -34,10 +34,26 @@ public DepositExtractor(IFederatedPegSettings federatedPegSettings, Network netw
             this.opReturnDataReader = opReturnDataReader;
         }
 
+        private static Dictionary<string, List<IDeposit>> DepositsToInject = new Dictionary<string, List<IDeposit>>()
+        {
+            { "CirrusRegTest", new List<IDeposit> { 
+                new Deposit(0x1 /* Tx of deposit being redone */, DepositRetrievalType.Small, new Money(10, MoneyUnit.BTC), "qZc3WCqj8dipxUau1q18rT6EMBN6LRZ44A", DestinationChain.STRAX, 12500, 0) } }
```

Comment:
What transaction id will be injected here? Is it the one that was originally done incorrectly?

Reply:
If that is the case... I'm wondering if we shouldn't check whether it actually exists first?

Also, what stops quorum amount of Multisig members from amending the code and injecting deposits as they please?

Reply:
Yes, and you are mirroring my thoughts on this. We could check if it exists as the core principle here is that we should only process deposits that have bad addresses and have not been processed before. However it does seem a bit unnecessary as we would know all those things before adding it to the list.

> Also, what stops quorum amount of Multisig members from amending the code and injecting deposits as they please?

I don't think it really relates to quorum but more our ability to modify the code base. In that case the answer would be the same as for any code change we make. The users vote by downloading and using those code (or not).